spotbugs/etc/messages.xml

```diff
@@ -7025,6 +7033,29 @@ just use the constant. Methods detected are:
     ]]>
       </Details>
   </BugPattern>
+  <BugPattern type="PA_PUBLIC_ATTRIBUTES">
+    <ShortDescription>Field is both public and written by a method</ShortDescription>
+    <LongDescription>Field {1} is both public and written by a method</LongDescription>
+    <Details>
+   <![CDATA[
+    <p>
+    SEI CERT rule OBJ01-J requires that accessibility of fields must be limited:
+    https://wiki.sei.cmu.edu/confluence/display/java/OBJ01-J.+Limit+accessibility+of+fields
```

Comment:
`<Details>` can use HTML, so consider using an `<a>` element here to guide readers to the page.